CardsCashCasino/CardCashCasinoGame.cs

```diff
@@ -81,7 +128,15 @@ protected override void Update(GameTime gameTime)
         /// </summary>
         protected override void Draw(GameTime gameTime)
         {
-            GraphicsDevice.Clear(Color.CornflowerBlue);
+            GraphicsDevice.Clear(Color.Green);
+
+            _spriteBatch!.Begin(samplerState: SamplerState.PointClamp);
+            // same for the main menu
+            if (_blackjackManager.IsPlaying)
+                _blackjackManager.Draw(_spriteBatch!);
+            // same for texas hold em
```

Comment:
have we fleshed out the draw functionality for texas hold em yet?

Reply:
Yes. It's in this PR,, but I can't get anything to show to the screen. More than likely just missing a ton of logic for it.

Reply:
Oh. I'm an idiot. That's why it isn't drawing. I'll add that line in now.